cdap-api/src/main/java/co/cask/cdap/api/customaction/AbstractCustomAction.java

```diff
@@ -58,6 +61,7 @@ protected void setProperties(Map<String, String> properties) {
   }
 
   @Override
+  @TransactionPolicy(TransactionControl.IMPLICIT)
```

Comment:
E.g. we can have it at the `ProgramLifecycle` level with `implicit` and only `Worker` and `CustomActionContext` overrides it to annotate with `explicit`.